pkgs/by-name/ne/neohtop/package.nix

```diff
@@ -0,0 +1,78 @@
+{
+  rustPlatform,
+  cargo-tauri,
+  nodejs,
+  jq,
+  stdenv,
+  fetchNpmDeps,
+  zsh,
+  fetchFromGitHub,
+  npmHooks,
+  lib,
+  openssl,
+}:
+
+let
+  tauri = cargo-tauri;
+
+in
+rustPlatform.buildRustPackage rec {
+  name = "neohtop";
+  pname = name;
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "abdenasser";
+    repo = "neohtop";
+    rev = "v${version}";
+    hash = "sha256-23r/oBR4zWmeSVa5QaJ2wJOwzwmqYVhMKOpaEABP7Qs=";
+  };
+  buildInputs = [
+    openssl
+  ];
```

Comment:
This file, in general, needs a lot more spaces between attributes to better group things that are related (e.g. `cargoRoot` and `cargoLock` put together, separated from `npmDeps`)